docs/src/main/sphinx/connector.rst

```diff
@@ -34,6 +34,7 @@ from different data sources.
     Pinot           <connector/pinot>
     PostgreSQL      <connector/postgresql>
     Prometheus      <connector/prometheus>
+    Ranger          <connector/ranger>
```

Comment:
This would move [here](https://github.com/trinodb/trino/blob/master/docs/src/main/sphinx/security.rst#access-control).

cc: @mosabua